src/Chapter2.hs

```diff
@@ -731,7 +764,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains n xs = filter (\x -> n `elem` x) xs
```

Comment:
Nice solution! I'm just sharing a slightly shorter version :slightly_smiling_face: 
In here I've used eta-reduction to get rid of `xs`. And I replace lambda with partial application of the `elem` function. It might look a bit tough to decipher at first, but the more you practice, the more you get used to such notation.

```suggestion
contains n = filter (elem n)
```

Reply:
Totally missed that I can eta reduce the filter fn, thanks for pointing that out!